src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.spec.ts

```diff
@@ -49,16 +68,124 @@ describe('PagesWalletsMyWalletsComponent', () => {
           expect(component.myWalletsData.data).toEqual([new MyWallet(walletResp)]);
         });
       });
+
+      describe('error', () => {
+        it('should set error state to false when get my wallets throws fail', () => {
+          walletsSubject.error('error');
+
+          expect(component.myWalletsData.data).toEqual(null);
+          expect(component.myWalletsData.hasError).toEqual(true);
+        });
+
+        it('should display notification container when get my wallets throws fail', () => {
+            const debugElement = fixture.debugElement;
+            walletsSubject.error('error');
+
+            fixture.detectChanges();
+
+            const notificationContainer = debugElement.query(By.css('#notificationContainer'));
+            expect(notificationContainer.nativeElement).toBeTruthy();
+          });
+      });
     });
   });
 
   describe('createWallet', () => {
-    it('should create new wallet', () => {
-       component.myWalletsData.data = [];
-       walletsSubject.next([new MyWallet(walletResp)]);
+    it('success', () => {
+      component.myWalletsData.data = [];
```

Comment:
Considering that both those tests (success and error) are setting this data property to empty array you can place it in beforeEach for this whole describe block.